components/clamav/files/etc/clamav-milter.conf.example-v0.98.1

```diff
@@ -0,0 +1,293 @@
+##
```

Comment:
Why can't we use example from package, why should we store it? If necessary, it can be patched. Delivering own file can be suitable if patch is too big.